cql3/restrictions/statement_restrictions.hh

```diff
@@ -360,18 +360,10 @@ public:
         return _partition_level_filter;
     }
 
-    // Returns any filter that needs to be applied to the clustering key. If the clustering key restriction is translated
+    // Returns any filter that needs to be applied to the clustering rows. If one of the column restrictions is translated
```

Comment:
Maybe say **each clustering row** instead of "the clustering rows". Maybe also here worth mentioning in the comment the specific column types this can be: clustering column and regular column. Unless it can be something else?

Reply:
It can be something else, though not with the current grammar.

Reply:
I added more explanation to the covering letter.

Reply:
Ok, I agree.

So should I merge this version, or do you still want to make some changes to the comments (or anything else) and I should wait a bit more?

Reply:
I'll enhance a few comments.

Note: if we're clever we can factor out static column references even if they're not part of a conjunction:

WHERE (static_col + regular_col) > 3

can be translated to

partition level filter: static_col IS NOT NULL

clustering row level filter: (static_col + regular_col) > 3

Although, I don't expect it's worthwhile.